src/Metadata/Resource/Factory/CachedResourceNameCollectionFactory.php

```diff
@@ -54,12 +54,8 @@ public function create() : ResourceNameCollection
             return $resourceNameCollection;
         }
 
-        try {
-            $cacheItem->set($resourceNameCollection);
-            $this->cacheItemPool->save($cacheItem);
-        } catch (CacheException $e) {
-            // do nothing
-        }
+        $cacheItem->set($resourceNameCollection);
```

Comment:
IMHO there is no reason to catch a `CacheException` exception here because `CacheItemPoolInterface::save()` and `CacheItemInterface::set()` are not planning to throw an exception in their method definitions.